doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,488 @@
+* Integrated Apple HomeKit ADK v5.3 and added development support for nRF52832 and nRF52833 for Bluetooth LE HomeKit accessories.
+* Zigbee Network Co-Processor (NCP) design is now supported for production for nRF52833 and nRF52840.
+* The application layer part of ZBOSS Zigbee stack is available in source code now.
+* New sample :ref:`modem_shell_application` can be used to test various connectivity features.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.6.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` for more information.
+
+Supported modem firmware
+************************
+
+This version of the |NCS| has been tested with the following modem firmware for cellular IoT applications:
+
+* mfw_nrf9160_1.3.0
+* mfw_nrf9160_1.1.4
+
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+nRF9160
+=======
+
+* Added:
+
+  * :ref:`lib_modem_jwt` library, which provides an API to obtain a JSON Web Token (JWT) from the modem.  Functionality requires modem firmware v1.3.0 or higher.
+
+  * :ref:`lib_modem_attest_token` library:
+
+    * The library provides an API to get an attestation token from the modem.
+    * Functionality requires modem firmware v1.3.0 or higher.
+
+  * :ref:`mod_memfault` module and integration on |NCS| for nRF9160-based devices.
+
+    * Integration of Memfault SDK into |NCS|.
+
+  * :ref:`memfault_sample` sample implementing :ref:`mod_memfault` module and showing how to use Memfault SDK functionality in an application.
+
+  * :ref:`modem_shell_application` enables to test various connectivity features such as link handling, TCP/IP connections, data throughput (curl and iPerf3), SMS, GNSS, FOTA and PPP.
+
+  * :ref:`lib_nrf_cloud_pgps` library:
+
+    * The library adds P-GPS (Predicted GPS) support to the :ref:`lib_nrf_cloud` library.
+
+  * :ref:`sms_sample` demonstrates how you can send and receive SMS messages with your nRF9160-based device using the :ref:`sms_readme`.
+
+  * :ref:`pdn_sample` demonstrates how to create and configure a Packet Data Protocol (PDP) context, activate a Packet Data Network connection, and receive events on its state and connectivity using the :ref:`pdn_readme` library.
+
+  * :ref:`pdn_readme` library can be used to manage Packet Data Protocol (PDP) contexts and PDN connections.
+
+* Updated:
+
+  * :ref:`lib_nrf_cloud` library:
+
+    * Added cellular positioning support to :ref:`lib_nrf_cloud_cell_pos`.
+    * Added Kconfig option :option:`CONFIG_NRF_CLOUD_CELL_POS` to obtain cell-based location from nRF Cloud instead of using the modem's GPS.
```

Comment:
This could be moved to 'Added' section: 

```suggestion
 *  :ref:`lib_nrf_cloud_cell_pos` library: 
    * Added cellular positioning support to :ref:`lib_nrf_cloud` library.
    * Added Kconfig option :option:`CONFIG_NRF_CLOUD_CELL_POS` to obtain cell-based location from nRF Cloud instead of using the modem's GPS.
```

Reply:
I will leave as is. The Added section contains only completely new libs and samples.